spk/duplicity/Makefile

```diff
@@ -49,4 +33,30 @@ SPK_COMMANDS = bin/duply bin/duplicity
 SPK_USR_LOCAL_LINKS  = bin:env/bin/duplicity
 SPK_USR_LOCAL_LINKS += bin:bin/duply
 
+# [cryptography]
+DEPENDS += cross/openssl3
```

Comment:
cryptography comment above is irrelevant.

Reply:
Don't we need the `cross/openssl3` for the cryptography wheel to build? That was the reason for the comment.

Reply:
Its just me being picky, it's not an actual issue which is why I marked it as approved.  The `[bla]` tags were really meant to tell the special hooks of each wheels and openssl3 is so commonly used elsewhere that to me it's not that much specific to cryptography.  In this specific context although it may well be... Long-story short, feel free to not bother with that.